firepit/pgstorage.py

```diff
@@ -149,18 +151,25 @@ def __init__(self, dbname, url, session_id=None):
 
         logger.debug("Connection to PostgreSQL DB %s successful", dbname)
 
+    def _create_firepit_common_schema(self):
+        try:
+            self._execute(f'CREATE SCHEMA IF NOT EXISTS "firepit_common";')
```

Comment:
You don't need an f-string here.
If the schema *does* exist, will this raise an exception?  We only need to create this once, so we need a way to check if the schema already exists and don't bother creating it if it does.